include/grpc++/impl/codegen/async_stream.h

```diff
@@ -165,18 +168,20 @@ class ClientAsyncReader final : public ClientAsyncReaderInterface<R> {
   static ClientAsyncReader* Create(ChannelInterface* channel,
                                    CompletionQueue* cq, const RpcMethod& method,
                                    ClientContext* context, const W& request,
-                                   void* tag) {
+                                   bool start, void* tag) {
```

Comment:
Please update the comment above to include `start`.
We may want to add an assert to say when you set `start` to true you have to pass in a `nullptr` as tag to prevent mis-use?

Reply:
I added the comments and asserts requested, plus also asserts on each of the Read/Write/Finish functions to make sure that started_ has already happened by that time (either through the constructor or StartCall)